R/brdg_hierarchical.R

```diff
@@ -308,7 +308,6 @@ pier_summary_hierarchical <- function(cards,
     # this ensures the correct order when there are 10+ groups
```

Comment:
@edelarua can you take a look at this?

- I think we may just need to replace the preceding `|>` with `%>%` to put the input data frame in thte second argument.
- If we don't already have a test for AE tables with 10+ by levels, can we add one?

Reply:
With both `|>` or `%>%`, the LHS comes in as the first argument, and `cards_no_attr |> ...` as the second. The dot comes in third, and is mapped to `copy`, which currently is never touched in dplyr and hence never errors.

Reply:
Using the dot with `%>%` would change the default location of the argument placement. From the `%>%` help file:

<img width="798" alt="image" src="https://github.com/user-attachments/assets/165583a5-828d-4117-aaa7-66217acbd1ec">

Reply:
Oh, right, I forgot about that. Can we use `y = _` with the base pipe too?

Reply:
How about storing the RHS in a variable?